README.md

```diff
@@ -429,6 +429,8 @@ You can now override the context portForward default address configuration by se
       logoless: false
       # Set to true to hide K9s crumbs. Default false
       crumbsless: false
+      # Set to true to suppress the K9s splash screen on start. Default false.
```

Comment:
We should add a caution here i.e if there are connectivity lags or large clusters are involved there could be no resources showing until the caches are provisioned...

Reply:
Done!